### ISSUE TITLE
EntityDamageByEntityEvent Error

### ISSUE THREAD

**Issue by Momonche** [mcmonkeyprojects/Sentinel#309]
The output of command `/version` on my server is: **git-Paper-173 (1.14.4)**
The output of command `/version citizens` on my server is: **2.2.25-SNAPSHOT (build 1743)**
The output of command `/version sentinel` on my server is: **1.9.5 (build 283)**
A pastebin.com link to my (full!) `Sentinel/config.yml` file is: **https://pastebin.com/yq0heLkg**

Hello @mcmonkey4eva 
I'm trying to have a stable 1.14.4 server so any error can be usefull and I have this error, maybe you can help me ?
https://pastebin.com/burssyRQ

**Comment by mcmonkey4eva** [comment]
- You misformatted your post a bit (the actual text of the post was made invisible!), I've fixed it for you.

- If I understand the sequence of events that caused this error... a Sentinel fired an arrow, THEN died, THEN the arrow landed in a target. That's ... impressive.

- I've fixed the error (assuming my interpretation above to be correct) for build 285+ https://ci.citizensnpcs.co/job/Sentinel/
